TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -274,12 +285,12 @@ namespace Plugin {
         return ++counter;
     }
 
-    uint32_t TextToSpeechImplementation::Speak(const string text,uint32_t &speechid,Exchange::ITextToSpeech::TTSErrorDetail &ttsStatus)
+    uint32_t TextToSpeechImplementation::Speak(const string text,const string callsign,uint32_t &speechid,Exchange::ITextToSpeech::TTSErrorDetail &ttsStatus)
```

Comment:
If we resort to updating the API, it would be intuitive to make the callsign the first argument and the text as the next

Reply:
ok